### PR TITLE
[14.0][MIG] stock_quant_view_reservation

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
@Cedric-Pigeon

### CODE REVIEW

File: stock_quant_view_reservation/models/stock_quant.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from odoo import models
+
+
+class StockQuant(models.Model):
+
+    _inherit = "stock.quant"
+
+    def action_view_reservations(self):
+        self.ensure_one()
+        action = self.env.ref("stock.stock_move_line_action").read([])[0]
```
Comment (rousseldenis) [inline stock_quant_view_reservation/models/stock_quant.py]: Use _for_xml_id() instead

File: stock_quant_view_reservation/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Cédric Pigeon <cedric.pigeon@acsone.eu>
```
Comment (rousseldenis) [inline stock_quant_view_reservation/readme/CONTRIBUTORS.rst:1]: You can put yourself here

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Thanks for this.

**Review by rousseldenis — APPROVED** [review]

**Review by acsonefho — APPROVED** [review]

File: stock_quant_view_reservation/views/stock_quant.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="stock_quant_tree_view">
+        <field name="name">stock.quant.tree (in stock_quant_view_reservations)</field>
+        <field name="model">stock.quant</field>
+        <field name="inherit_id" ref="stock.view_stock_quant_tree_editable" />
```
Comment (rousseldenis) [inline stock_quant_view_reservation/views/stock_quant.xml]: @xavier-bouquiaux Tested functionally and the button does not appear.

IMHO You should also add it in normal tree view:

https://github.com/odoo/odoo/blob/14.0/addons/stock/views/stock_quant_views.xml#L152

File: stock_quant_view_reservation/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Go to Inventory > Master Data > Products and click "On hand" smart button.
+On all quants you can a button with an eye icon. It opens the move line reserving some quantities on the location.
```
Comment (rousseldenis) [inline stock_quant_view_reservation/readme/USAGE.rst:2]: IMHO these sentences should be improved

**Review by HviorForgeFlow — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**